test/CMakeLists.txt

```diff
@@ -111,15 +115,16 @@ function(bufr_add_test_preAPX TESTNAME EXENAME)
            COMMAND ${CMAKE_BINARY_DIR}/bin/test_wrapper_OUT.sh ${EXENAME} "Y")
 endfunction()
 
-# test all "4" (4 byte real, 4 byte int) DYNAMIC_ALLOCATION builds
-list(APPEND test_kinds "4_DA")
-# test all "4" (4 byte real, 4 byte int) STATIC_ALLOCATION builds, except OUT_3 and OUT_4 as noted below
-list(APPEND test_kinds "4")
+list(APPEND test_kinds_4 "4_DA")
+list(APPEND test_kinds_4 "4")
```

Comment:
could be condensed into `list(APPEND test_kinds_4 "4_DA" "4")`

Reply:
Noted